toolkits/global/packages/global-breadcrumb/scss/10-settings/default.scss

```diff
@@ -0,0 +1,6 @@
+$breadcrumb-font-family: $font-family-sans;
+$breadcrumb-font-size-sm: $font-size-sm;
```

Comment:
`$font-size-sm` is from the springer context and won't be available here. I would just whatever pixel value this represents here.